client_integration_test.go

```diff
@@ -0,0 +1,191 @@
+package notify_test
+
+import (
+	"os"
+	"testing"
+
+	notify "github.com/govau/notify-client-go"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var apiKey string
+var smsTemplateId string
+var emailTemplateId string
+var phoneNumber string
+var emailAddress string
+var client *notify.Client
+
+func setup(t *testing.T) {
```

Comment:
At the very least I think setup can return a `client` rather than setting up a global one. Should be reset every test anyway I think.